salt/runners/reactor.py

```diff
@@ -67,8 +67,8 @@ def list_(saltenv="base", test=None):
 
         __jid_event__.fire_event({"key": master_key}, "salt/reactors/manage/list")
 
-        results = sevent.get_event(wait=30, tag="salt/reactors/manage/list-results")
-        reactors = results["reactors"]
+        results = sevent.get_event(wait=30, tag="salt/reactors/manage/list-results")    
+        reactors = results.get('reactors', None)
```

Comment:
```suggestion
        reactors = results.get('reactors')
```
This does exactly the same.